### DISCUSSION TITLE
how to uninstall

### DISCUSSION THREAD

**Discussion by sprit77** [jely2002/youtube-dl-gui#465] [Q&A] [answered]
how can I uninstall this from mac? Thanks for helping ::)

**Comment by jely2002** [comment, marked as answer]
To uninstall from Mac you can follow these steps:

1. Remove the `Open Video Downloader.app` file from your `Applications` folder.
2. If you also don't want to keep any settings, you can remove the folder at `/Users<youruser>/Library/Application Support/com.jelleglebbeek.youtube-dl-gui`. That folder contains the app settings and other files it needs. If you ever decide to re-install this folder will just be regenerated.

Hope this helps other people in the future.
